### DISCUSSION TITLE
Synced Lyrics doesn't work for me?

### DISCUSSION THREAD

**Discussion by hattbsdk** [pear-devs/pear-desktop#2360] [Q&A]
Hey y'all - I'm not a coder or very fluent in very technical aspects of this app, but I saw the synced lyrics plugin and tried using it (restarted the app) but it doesn't seem to be working for me. Am I doing something wrong?

![image](https://github.com/user-attachments/assets/e37d5288-78ee-412d-8dfa-7eb3b9251d10)

**Comment by ArjixWasTaken** [comment]
That is odd, it is working for me
![image](https://github.com/user-attachments/assets/eab5b4ea-4781-436c-8a6a-1a4542369992)

**Comment by BlackRefraction** [comment]
Hey bro, I'm actually seeing the exact same thing. first I thought the song was too obscure but it doesn't load ANY lyrics, the unsynced ones work fine but when I enable the synced lyrics it just says no lyrics found, does anyone know how to fix it? Tried a reboot and re-install, didn't work.

![YTM](https://github.com/user-attachments/assets/b145fc1f-7d7f-42ba-9a90-d4747541a2ae)

**Reply by ArjixWasTaken** [reply]
Please wait for the upcoming version of youtube-music.
